.github/workflows/tests.yml

```diff
@@ -396,12 +396,12 @@ jobs:
 
   windows-native:
     name: Windows (Native)
-    runs-on: windows-2019
+    runs-on: windows-2022
```

Comment:
It's possible that we want to selectively keep the old `windows-2019` actions, too, given [this quote](https://github.blog/changelog/2021-11-16-github-actions-windows-server-2022-with-visual-studio-2022-is-now-generally-available-on-github-hosted-runners/):

> The Windows Server 2022 runner image has [different tools and tool versions](https://github.com/actions/virtual-environments/issues/3949) than Windows Server 2019.

Reply:
Sure. Unlike macOS 12 vs macOS 14, which has different CPU architecture, I expect Windows 2019 vs 2022 to only be relevant for installation failures. So, it doesn't have to run on every PR, but once in a while wouldn't hurt.